{"name": "test_update_name_without_auth", "status": "passed", "description": "Тест: Попытка обновления имени без авторизации", "steps": [{"name": "Отправляем запрос на обновление имени без авторизации", "status": "passed", "start": 1741316123906, "stop": 1741316124650}, {"name": "Проверяем, что возвращается ошибка 401", "status": "passed", "start": 1741316124650, "stop": 1741316124650}], "start": 1741316123906, "stop": 1741316124650, "uuid": "214ff0d6-85b2-4cc3-8a8b-30f9091daa22", "historyId": "2dcd036c3ed6a668a1fda3e4f6ff1c2f", "testCaseId": "2dcd036c3ed6a668a1fda3e4f6ff1c2f", "fullName": "tests.test_create_update_user.TestUserEndpoint#test_update_name_without_auth", "labels": [{"name": "severity", "value": "critical"}, {"name": "epic", "value": "Пользователи"}, {"name": "feature", "value": "Работа с профилем"}, {"name": "story", "value": "Обновление имени без авторизации"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_update_user"}, {"name": "subSuite", "value": "TestUserEndpoint"}, {"name": "host", "value": "DESKTOP-UNG0SPG"}, {"name": "thread", "value": "35156-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_update_user"}]}